README.md

```diff
@@ -39,3 +39,37 @@ This will create a container called `fedora-toolbox-<version-id>`.
 [user@hostname ~]$ toolbox enter
 ⬢[user@toolbox ~]$
 ```
+
+## Install dependencies, build and install toolbox
+
+### Install dependencies:
```

Comment:
I wonder if we should also list out the run-time dependencies. eg., currently we require Podman >= 1.4.0 but I don't think that's clearly mentioned anywhere other than in the Git commit messages and Fedora package.

Reply:
That sounds reasonable. We can use this section of README where we track this information.